apps/website/src/components/VideoCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { Heading, Card, CardBody, AspectRatio, HStack } from "@chakra-ui/react"
+
+export type VideoCardProps = {
+    videoUrl: string
+    title: string
+}
+
+export default function VideoCard({ videoUrl, title }: VideoCardProps) {
+    return (
+        <Card
+            bg={"transparent"}
```

Comment:
It looks like the background color could change. It should be a component property.